Resources/Prototypes/Nyanotrasen/metempsychoticHumanoids.yml

```diff
@@ -0,0 +1,15 @@
+- type: weightedRandom
+  id: MetempsychoticHumanoidPool # Species prototypes
```

Comment:
Are these automatically checked to see if all prototypes are valid? If not, this'll require a test to be added

Reply:
![image](https://github.com/DeltaV-Station/Delta-v/assets/16548818/59182ecd-3e59-4793-98c1-426c3c324605)
Yes these are automatically validated by MetempsychoticMachineSystem.

Reply:
By creating errors if they aren't valid yes. Nothings stopping you from comitting invalid prototypes